### PR TITLE
HF Token Fixes

### PR DESCRIPTION
**Pull request by saileshd1402**
Changes:
- Hide HF Token in the generate command
- Remove HF Token dependency in run command

### CODE REVIEW

**Review by gavrishp — DISMISSED** [review]

File: llm/kubeflow_inference_run.py [hunk 1]
```diff
@@ -381,11 +380,40 @@ def execute(params: argparse.Namespace) -> None:
...
+        "model directory",
+        is_dir=True,
+    )
+
     if not model_params["is_custom"]:
         if not model_info["repo_version"]:
             model_info["repo_version"] = model_params["repo_version"]
         model_info["repo_id"] = model_params["repo_id"]
-        model_info["repo_version"] = check_if_valid_version(model_info, mount_path)
+        model_info["repo_version"] = update_repo_version(model_info, mount_path)
```
Comment (gavrishp) [inline llm/kubeflow_inference_run.py:393]: Might have missed last time. `update_repo_version()` with a return does not make sense. As `model_info` is passed by ref, don't need to return?

Else the func name should be `set_repo_version()`

File: llm/kubeflow_inference_run.py [hunk 2]
```diff
@@ -381,11 +380,40 @@ def execute(params: argparse.Namespace) -> None:
...
 
+    check_if_path_exists(
+        os.path.join(mount_path, model_info["model_name"]),
+        "model directory",
+        is_dir=True,
+    )
+
     if not model_params["is_custom"]:
         if not model_info["repo_version"]:
             model_info["repo_version"] = model_params["repo_version"]
```
Comment (gavrishp) [inline llm/kubeflow_inference_run.py:391]: also setting first then updating immediately? Doesn't look clean

File: llm/kubeflow_inference_run.py [hunk 3]
```diff
@@ -381,11 +380,40 @@ def execute(params: argparse.Namespace) -> None:
...
         model_info["repo_id"] = model_params["repo_id"]
-        model_info["repo_version"] = check_if_valid_version(model_info, mount_path)
+        model_info["repo_version"] = update_repo_version(model_info, mount_path)
+
+    check_if_path_exists(
+        os.path.join(
+            mount_path,
```
Comment (gavrishp) [inline llm/kubeflow_inference_run.py:397]: have a variable / helper method with path_to_model_dir set. No need to add them comma separated multiple times

Comment (saileshd1402) [reply llm/kubeflow_inference_run.py]: created a new function

File: llm/kubeflow_inference_run.py (hunk 2 above)
Comment (saileshd1402) [reply llm/kubeflow_inference_run.py]: changed this to set it directly in the function

File: llm/kubeflow_inference_run.py (hunk 1 above)
Comment (saileshd1402) [reply llm/kubeflow_inference_run.py]: fixed this

**Review by gavrishp — APPROVED** [review]

**Pull request merged**